src/server/plugins/initialiseSession/helpers.ts

```diff
@@ -63,6 +63,8 @@ export function verifyToken(decodedToken) {
     })
     return { isValid: true }
   } catch (err) {
+    console.log(err)
```

Comment:
this is already logged further up the chain, can we remove please

Reply:
Well spotted, shouldn't have been left in. Will remove